### PR TITLE
[docs] - Clean up Celery execution guide

### PR DESCRIPTION
**Pull request by erinkcochran87**
## Summary & Motivation

This PR cleans up the copy and formatting of the Executing on Celery OSS guide.

## How I Tested These Changes

👀 , reading

### CODE REVIEW

File: docs/content/deployment/guides/celery.mdx [hunk 1]
```diff
@@ -36,40 +54,42 @@ def parallel_job():
...
 
 In production, more configuration is required.
 
-### 1. Persistent run and event log storage
+### Step 1: Configure persistent run and event log storage
+
+First, you'll ensure that appropriate persistent run and event log storage, e.g., `PostgresRunStorage` and `PostgresEventLogStorage`, are configured on your [Dagster instance](/deployment/dagster-instance) (via `dagster.yaml`). This allows the webserver and workers to communicate information about the run and events with each other.
 
-First, ensure that appropriate persistent run and event log storage, e.g., **`PostgresRunStorage`** and **`PostgresEventLogStorage`**, are configured on your Dagster instance, so that the webserver and the workers can communicate information about the run and events. You can do this by adding a block such as the following to your `dagster.yaml` (by default, Dagster will look for this file at `$DAGSTER_HOME/dagster.yaml` when the `DAGSTER_HOME` environment variable is set):
+Add a block like the following to your project's `dagster.yaml`:
```
Comment (johannkm) [inline docs/content/deployment/guides/celery.mdx:92]: can leave this but ideally we could just link to a page about configuring postgres storage

**Review by johannkm — APPROVED** [review]
Nice!

Could work in a link on this page to our dagster-celery-k8s docs https://docs.dagster.io/deployment/guides/kubernetes/deploying-with-helm-advanced

File: docs/content/deployment/guides/celery.mdx [hunk 2]
```diff
@@ -82,44 +102,83 @@ storage:
...
+To monitor celery workers and queues, you can use Celery's [Flower](https://flower.readthedocs.io/en/latest/) tool. This can be useful in understanding how workers interact with the queue.
 
-Celery also includes the excellent [Flower](https://flower.readthedocs.io/en/latest/) tool for monitoring your Celery workers and queues. This can be very useful in understanding how workers interact with the queue.
+### Broker and backend
 
-## Broker and backend
+`dagster-celery` has been tested using the RabbitMQ broker and default RPC backend.
 
-At the moment, dagster-celery is tested using the RabbitMQ broker and the default RPC backend. At least one team is running dagster-celery using SQS as the broker, and we intend to expand the scope of broker/backend matrix testing, probably starting with the Postgres backend.
+<!-- At least one team is running dagster-celery using SQS as the broker, and we intend to expand the scope of broker/backend matrix testing, probably starting with the Postgres backend. -->
```
[the remark this answers is not in the document]
Comment (erinkcochran87) [reply docs/content/deployment/guides/celery.mdx:184]: Right? I commented it out hoping someone would have info, but this definitely struck me as weird. I'm inclined to take it out.

[the remark this answers is not in the document]
Comment (johannkm) [reply docs/content/deployment/guides/celery.mdx:184]: I think this goes back to the early days 🙂

[the remark this answers is not in the document]
Comment (erinkcochran87) [reply docs/content/deployment/guides/celery.mdx:184]: Probably - I'm finding so many fun things like this as I clean up lol

**Pull request merged**